js/modules/offline-exporting.src.js

```diff
@@ -6,11 +6,54 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * Download options for offline exporting.
+ *
+ * @typedef Highcharts.OfflineExportingDownloadOptionsObject
```

Comment:
I believe these are the same options that are available for the online exporting, so they should probably use the same typedef.